crates/dataflow-jit/src/codegen/mod.rs

```diff
@@ -1399,6 +1474,79 @@ impl<'a> CodegenCtx<'a> {
         self.add_expr(expr_id, value, value_ty, None);
     }
 
+    fn cast(&mut self, expr_id: ExprId, cast: &Cast, builder: &mut FunctionBuilder<'_>) {
```

Comment:
notice that in the SQL compiler all casts are implemented manually as part of the runtime sql library. The numeric ones are relatively homogeneous, but there are some really weird ones in SQL. If we could support the sql library you would not need to implement casts. The generated code from the SQL compiler invokes these functions, which are named in some homogeneous way. Please see casts.rs

Reply:
It'd be a lot better if the SQL compiler emitted a cast instruction, otherwise we either pay big costs at every cast operation or I have to manually write a bunch of `if function.name == "cast_to_i64_i32"`

Reply:
This has been fixed in https://github.com/vmware/sql-to-dbsp-compiler/pull/105